docs/release_process.md

```diff
@@ -0,0 +1,269 @@
+
+Then, you can push your local `main` to heroku:
+
+* `git push heroku main`
+
+We can also push un-merged branches:
+
+* `get push heroku change-1:main`
+
+
+## Release to Stage
+Every tag pushed to GitHub is automatically deployed to the [Stage][stage]
+server. The standard practice is to create a tag from `main` every Tuesday at
+the end of the day, and to name the tag with `YYYY-MM-DD` [CalVer][calver]
+syntax. This tag will include only the changes that have been merged to `main`.
+E.g., the following `2022.08.02` tag includes only `change-1` and `change-2`.
+
+```mermaid
+%%{init: { 'theme': 'base', 'gitGraph': {'rotateCommitLabel': true} } }%%
+    gitGraph
+       commit
+       branch change-1
+       commit
+       commit
+       checkout main
+       branch change-2
+       commit
+       checkout main
+       merge change-1
+       branch change-3
+       commit
+       commit
+       checkout main
+       merge change-2 tag: "2022.08.02"
+       checkout change-3
+       commit
+       commit
+       checkout main
+```
+
+## Release to Prod
+We leave the tag on [Stage][stage] for a week so that we (and especially QA)
+can check the tag on GCP infrastucture before we deploy it to production. To
+deploy the tag to production:
+
+1. [Make a release on GitHub][github-new-release] for the tag.
+   * Use the "Generate release notes" button!
+2. File an [SRE ticket][sre-board] to deploy the tag to [Prod][prod].
+   * Include a link to the GitHub Release
+
+## Stage-fixes
+Ideally, every change can ride the regular weekly release "trains". But
+sometimes we need to make and release changes before the regularly scheduled
+release.
+
+### "Clean `main`" flow
+If a bug is caught on [Stage][stage] in a tag that is scheduled to go to
+[Prod][prod], we need to fix the bug before the scheduled prod deploy. If
+`main` is "clean" - i.e., nothing else has merged yet, we can use the regular
+GitHub Flow:
+
+1. Create a stage-fix branch from `main`:
+2. Make changes
+3. Create a pull request to `main`
+4. Address review comments
+5. Merge pull request
+6. Make and push a new tag
+
+```mermaid
+%%{init: { 'theme': 'base' } }%%
+    gitGraph
+       commit
+       branch change-1
+       commit
+       commit
+       checkout main
+       branch change-2
+       commit
+       checkout main
+       merge change-1
+       branch change-3
+       commit
+       commit
+       checkout main
+       merge change-2 tag: "2022.08.02"
+       checkout change-3
+       commit
+       commit
+       checkout main
+       branch stage-fix-1
+       commit
+       checkout main
+       merge stage-fix-1 tag: "2022.08.02.01"
+```
+
+### "Dirty `main`" flow
+If a bug is caught on [Stage][stage] in a tag that is scheduled to go to
+[Prod][prod], we need to fix the bug before the scheduled prod deploy. If
+`main` is "dirty" - i.e., other changes have merged, we can make the new tag
+from the stage-fix branch.
+
+1. Create a stage-fix branch from `main`:
```

Comment:
We should make the branch from the tag instead, which would make the `clean main` and `dirty main` flow the same.
```suggestion
1. Create a stage-fix branch from the tag `main`: `git branch stage-fix-2022.08.02 2022.08.02; git switch stage-fix-2022.08.02`
```